docs/telegram-usage.md

```diff
@@ -176,9 +177,24 @@ If Pair is set, then this pair will be added to the pairlist.
 Also supports multiple pairs, seperated by a space.
 Use `/reload_conf` to reset the blacklist.
 
-> Using blacklist `StaticPairList` with 2 pairs  
+> Using blacklist `StaticPairList` with 2 pairs
```

Comment:
Can you please disable the "trailing whitespace" extension for .md documents?
in vscode:
``` json
    "trailing-spaces.syntaxIgnore": [
        "markdown"
    ],
```

having 2 spaces at the end in markdown is allowed and generates a new line - which you removed with this change ...